pkgs/development/python-modules/csv2md/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "lzakharov";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-fyXM88hawO6Q9pwMgVtjXaPVEaP36TOmoYL0c7RFDZ4=";
+  };
+
```

Comment:
Could you please enable tests with `pythonImportsCheck` and `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages